osf/migrations/0137_add_fm_record_to_osfstorage_files.py

```diff
@@ -54,6 +54,4 @@ class Migration(migrations.Migration):
         ('osf', '0136_add_datacite_file_metadata_schema'),
     ]
 
-    operations = [
-        migrations.RunPython(add_records_to_files_sql, remove_records_from_files),
-    ]
```

Comment:
A follow-up comment since I was confused on the removal here during discussion, but I figured it out after testing this locally ...

For data backfill `RunPython()`s, in normal cases, we don't need to remove them since they will be gone during the clean migration squash. However, here we have to disable it because it needs the `datacite` population that we have moved to fixtures.